.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -25,8 +34,9 @@ body:
     attributes:
       label: Reproducible Code Example
       description: >
-        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)
-        on how to provide a minimal, copy-pastable example.
+        Please provide a simple example of code which can reproduce the bug.
+        [This guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)
+        explains how to provide a minimal, copy-pastable example.
```

Comment:
I really like this.

I feel there's something we could add in the handbook that could provide people with more context about why we need the minimal bug report, and how we use it. i.e., Make tests to ensure the problem never comes back.

That would, of course, link back to this guide above

Reply:
That guide has come directly from the pandas issue template, do you think it would be within the contribution guidelines to give advice for what types of contributions people can give e.g. code changes vs bug reports?